chart/samlet/values.yaml

```diff
@@ -16,5 +16,4 @@ samlet:
   #idpEndpoint: "https://example.com"
   idpEndpoint: ""
   # default validity time for issued secrets
-  #sessionDurattion: 1h
-  sessionDuration: ""
+  sessionDuration: "1h"
```

Comment:
One value should change to be consistent with the other. See https://github.com/AbsaOSS/samlet/blob/4e100d002a47fb14af20194f647f3540b60f53f1/README.md#L35

Reply:
@donovanmuller those are two different things. Controller option sets 'global' default. While .spec.sessionDuration can override this per object.

Reply:
Understood, thanks for the clarification 👍